cypress/e2e/api/EngageSphere.cy.js

```diff
@@ -0,0 +1,103 @@
+describe('EngageSphere API requests for the /customers Endpoint', () => {
+	context('Successful request', () => {
+		it('Successfully retrieves customers (e.g., checks for the 200 status code)', () => {
+			cy.api_getCustomers().then(({ status, body }) => {
+				expect(status).to.eq(200)
+				expect(body.customers).to.be.an('array').that.is.not.empty
+			})
+		})
+
+		it('Paginates the customer list correctly', () => {
+			cy.api_getCustomers({ page: 2, limit: 20, size: 'All' }).then(({ status, body }) => {
+				expect(status).to.eq(200)
+				expect(body).to.have.property('pageInfo').that.exist
+
+				const { pageInfo, customers } = body
+
+				expect(pageInfo.currentPage).to.eq('2')
+				expect(pageInfo.totalPages).to.eq(3)
+				expect(pageInfo.totalCustomers).to.be.gte(50)
```

Comment:
Aqui não deveria ser **less than or equal**? (`.to.be.lte(50)`)?

```suggestion
				expect(pageInfo.totalCustomers).to.be.lte(50)
```